crates/oxc_linter/src/utils/react.rs

```diff
@@ -112,6 +112,48 @@ pub fn object_has_accessible_child(node: &JSXElement<'_>) -> bool {
         || has_jsx_prop_lowercase(&node.opening_element, "children").is_some()
 }
 
+pub fn is_presentation_role(jsx_opening_el: &JSXOpeningElement) -> bool {
+    let Some(role) = has_jsx_prop(jsx_opening_el, "role") else {
+        return false;
+    };
+    let Some("presentation" | "none") = get_string_literal_prop_value(role) else {
+        return false;
+    };
+
+    true
+}
+
+// TODO: Should re-implement
+// https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/4c7e7815c12a797587bb8e3cdced7f3003848964/src/util/isInteractiveElement.js
+// with `oxc-project/aria-query` which is currently W.I.P.
+//
+// Until then, use simplified version by https://html.spec.whatwg.org/multipage/dom.html#interactive-content
+pub fn is_interactive_element(element_type: &str, jsx_opening_el: &JSXOpeningElement) -> bool {
```

Comment:
I'd like some advice on whether this makes sense?

Reply:
I think so?
did you have a particular concern about it?

Reply:
No, it was just a confirmation. 👀 As I wasn't sure about the progress of `oxc-project/aria-query`. (Should I wait for its implementation? etc.)